packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -76,6 +76,8 @@ export default declare((api, opts) => {
       if (t.isIdentifier(prop.key) && !prop.computed) {
         // since a key {a: 3} is equivalent to {"a": 3}, use the latter
         keys.push(t.stringLiteral(prop.key.name));
+      } else if (t.isTemplateLiteral(prop.key)) {
+        keys.push(t.templateLiteral(prop.key.quasis, prop.key.expressions));
```

Comment:
Since this is the same as doing `t.cloneNode(prop.key)`, you can use that function. Note that this shouldn't set `allLiteral` to `false` (because templates are always strings), so it shouldn't be merged with the last `else` block.

Reply:
Do you think something like check `isLiteral` and not `isTemplateLiteral`?